### ISSUE TITLE
No copies printed

### ISSUE THREAD

**Issue by DirkHaar** [OpenPrinting/cups-filters#451]
Reopened with new number, because it still isn't markes as bug. 

My Kyocera P5021cdw (wireless connection, colour laser, double-sided) prints multiple copies, everything is fine.

My Brother DCP-L2530DW (wireless connection, b/w laser, double-sided) doesn't print copies anymore,
when I need 10 copies I have to select printing e.g. from PDF viewer or LibreOffice print menu 10 times.
It doesn't matter how many pages the document contains or I f double-sided is chosen or what else.
"10 copies" is simply ignored - seems to be accepted - but only one is printed.
I have no idea how long this problem already exists (at least some months, but had no time to report it)
and wonder how this can be fixed.
address: dnssd://Brother%20DCP-L2530DW%20series._ipp._tcp.local/?uuid=e32...xxxxxxxxxxxxxxxx....88 (uuid manually replaced)
Make/model: Brother DCP-L2530DW series, driverless, cups-filters 1.27.4

Linux Mint 20.3 Una 64bit
Linux 5.13.0-27-generic
Cinnamon 5.2.7

If more information is needed I will provide it asap.

**Comment by Chandresh2702** [comment]
@DirkHaar  could you please provide the whole ppd file just make save it as a txt file and attach it here it could help in better understanding of problem

**Comment by Chandresh2702** [comment]
@DirkHaar  could you please try following solution 
update it to new version of cupsfliter 1.28.0 as there were some fixes for same type of issue #242 and #255 

And in issue #242 there were problem with a attribute in PPD which was false and chaging it to true solved that problem but it is missing in you ppd which was ```*cupsManualCopies: True```  but i cannot find the attribute at all in your PPD . could you please try to add this attribute in you ppd and and restart the cups once again and see if problem resolved and still exist.

and one more thing
Run the command
```cupsctl --debug-logging```
before Trying to print the multiple-copy job again. Then attach your /var/log/cups/error_log file here (remember that you have to rename the file to have a .txt extension so that you can attach it here). so that can see where the problem is

**Comment by DirkHaar** [comment]
Hi Chandresh, thanks for jumping in.
- My system has no open updates, so how do I update the cups-filter file manually?
- Further information I forgot: Linux Mint 20.3 Una, Linux 5.13.0-27-generic, Cinnamon 5.2.7
The PPD above is complete, besides 
*cupsManualCopies: True

The setting was
*cupsManualCopies: true

When wondering about no copies I found thatt and I did not understand why this should not be "false".
False does not work either.
I compared it to my Kyocera ppd  which has no such entry at all, so I removed the line
(giving the content above you consider incompete) which doesn't work too. 

Attached the log (without updating) for a test with three copies and 
*cupsManualCopies: false

[cupslog.txt](https://github.com/OpenPrinting/cups-filters/files/8177034/cupslog.txt)

**Comment by Chandresh2702** [comment]
@DirkHaar  thanks for Providing logs , but could you do one thing this may work or not.
Actually the issue I mentioned #242  has same type of issue with they tried both true/false but the actual problem was that these were case sensitive so true and false mean nothing there instead they have to use ```True / False``` after changing these attributes there problem fixed permanently.
Could you please try this once in the meantime i will go through youlogs and will try to find out if something else is creating the issue.

**Comment by Chandresh2702** [comment]
@DirkHaar  don't forget to restart cups after changing ppd. or just stop it before changing with 
```sudo systemctl stop cups```
and after changing start with 
```sudo systemctl start cups```
and then print

**Comment by Chandresh2702** [comment]
@DirkHaar  did you tried it

**Comment by DirkHaar** [comment]
TLDR;
Solution:
either update cups-filters to v1.28 OR use
`*cupsManualCopies: True`
in the ppd file.
